modules/data.atmosphere/R/met.process.R

```diff
@@ -316,15 +319,20 @@ met.process <- function(site, input_met, start_date, end_date, model,
                                        host = host, 
                                        overwrite = overwrite$standardize,
                                        ensemble_name = i)
+        is_standardized <- TRUE
+      } else {
+        is_standardized <- FALSE
```

Comment:
```suggestion
      } else {
        # No action taken. These ids will be dropped from ready.id
        id_stdized <- NULL
```

Reply:
It's possible this should be an error instead, with a message along the lines of "Only scales 'site' and 'regional' are currently supported". Thoughts?